proxy-data-atlas.conf.json

```diff
@@ -1,5 +1,5 @@
 {
-    "/api/*": {
+    "/api/**": {
```

Comment:
because the single `*` apparently only wildcards a single query path (folder). At least with only one `*` the proxy doesn't proxy my request to the permissions api because it has two slashes in it. With `**` everything gets correctly proxied. Not sure whether we never used more path segments before or angular changed something.